spec/support/command_helpers.rb

```diff
@@ -291,12 +293,12 @@ def reset_date
     @site.log "Reset date", level: :test
     command_list = build_command_list :M0104, :setDate, {
       securityCode: SECRETS['security_codes'][1],
-      year: Time.now.year,
-      month: Time.now.month,
-      day: Time.now.day,
-      hour: Time.now.hour,
-      minute: Time.now.min,
-      second: Time.now.sec
+      year: Time.now.utc.year,
```

Comment:
better to use Time.now.utc once and store in a local, then access year, month, etc in that.